personio-personnel-data-api-oa3.yaml

```diff
@@ -1749,7 +1749,7 @@ paths:
           - language: "php"
             code: "Auto-generated code snippets for this language may not be supported"
       responses:
-        "201":
+        "200":
```

Comment:
[Question]: Is this change willingly happening? Why are we changing the response code from `201` to `200` for a create endpoint?